sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveUtils.scala

```diff
@@ -198,6 +198,12 @@ private[spark] object HiveUtils extends Logging {
     .booleanConf
     .createWithDefault(true)
 
+  val HIVE_STATS_AUTOGATHER = buildConf("spark.sql.hive.stats.autogather")
+    .doc("When set to false, Hive client won't write default statitic metadata.")
+    .version("3.3.0")
```

Comment:
Should probably be 3.4.0.

Reply:
Do we really need to have a mapping Spark conf for each hive conf?

Reply:
> Do we really need to have a mapping Spark conf for each hive conf?

Removed